upgrade/1.0.11/Stage_0_Prerequisites.md

```diff
@@ -0,0 +1,181 @@
+# Stage 0 - Prerequisites and Preflight Checks
+
+> **NOTE:** CSM-1.0.1 or later is required in order to upgrade to CSM-1.0.11.
+>
+> **NOTE:** Installed CSM versions may be listed from the product catalog using the following command. This will sort a semantic version without a hyphenated suffix after the same semantic version with a hyphenated suffix, e.g. 1.0.11 > 1.0.11-beta.19.
```

Comment:
Why does the reader need to know the information in the 2nd sentence? This could use a sentence explaining why this information is necessary.

Reply:
removing the sentence